src/sssom/typehints.py

```diff
@@ -19,5 +24,33 @@
 class Metadata(NamedTuple):
     """A pair of a prefix map and associated metadata."""
 
-    prefix_map: PrefixMap
+    converter: Converter
     metadata: MetadataType
+
+    @property
+    def prefix_map(self):
+        """Get the bimap."""
+        return self.converter.bimap
+
+    @classmethod
+    def default(cls):
+        """Get default metadata."""
+        from .context import get_converter
+
+        return cls(
+            converter=get_converter(),
+            metadata=get_default_metadata(),
+        )
+
+
+def generate_mapping_set_id() -> str:
+    """Getnerate a mapping set ID."""
+    return f"{SSSOM_URI_PREFIX}mappings/{uuid.uuid4()}"
```

Comment:
Note that this logic seems duplicated with https://github.com/mapping-commons/sssom-py/pull/421/files#diff-fa45af370531fd5131bc80371b323932f856b104add1550cb1e7fe281f821133R14

Reply:
Resolved in 98122ac